activerecord/test/cases/enum_test.rb

```diff
@@ -892,10 +892,21 @@ def self.name; "Book"; end
     assert_respond_to book, :easy_to_read?
   end
 
+  test "enum labels as keyword arguments" do
+    klass = Class.new(ActiveRecord::Base) do
+      self.table_name = "books"
+      enum :status, active: 0, archived: 1
+    end
+
+    book = klass.new
+    assert_predicate book, :active?
+    assert_not_predicate book, :archived?
+  end
+
   test "option names can be used as label" do
     klass = Class.new(ActiveRecord::Base) do
       self.table_name = "books"
-      enum :status, { default: 0, scopes: 1, prefix: 2, suffix: 3 }
+      enum :status, default: 0, scopes: 1, prefix: 2, suffix: 3
```

Comment:
Originally this did test that if single keyword arguments is recognized as label values not options since label values are required arguments.

It works fine when two label values and keyword arguments are passed, but if users are unsure whether we can use a label with the same name as an option at the same time, I'll add an extra test. fe0fb9fdf04f638996e785d3e291f5fe8bc138a0